src/Components/FilterControls.js

```diff
@@ -21,32 +22,32 @@ class FilterControls extends Component {
   filterParades = () => { 
```

Comment:
here is our current function and how we're trying to accomplish filtering through all three options for seeing the parades.

`  filterParades = () => { 
    const { location, time, date } = this.state;
    const { parades } = this.props;
    let matchedIds = [];
    let matchedObjects = [];

    parades.forEach(parade => {
      if (!this.state.matchedIds.includes(parade.id) && 
      parade.location === location || 
      parade.dayOrNight === time ||
      parade.date === date)
      {
        matchedObjects.push(parade);
        matchedIds.push(parade.id);
      }
    });
    // this.setState({ matchedIds });
    console.log(matchedObjects)
    return matchedObjects
  }`